lib/pcr.c

```diff
@@ -211,6 +213,45 @@ bool pcr_print_pcr_struct(TPML_PCR_SELECTION *pcrSelect, tpm2_pcrs *pcrs) {
 }
 
 
+bool pcr_print_pcr_selection(TPML_PCR_SELECTION *pcrSelect) {
```

Comment:
I know this was just a copy paste from the existing code, but can we fix it up while you're at it.

Reply:
after consultation of the coding style, I assume you're referring to `pcrSelect` which I now renamed to `pcr_selections`